docs/api.asciidoc

```diff
@@ -20,7 +21,8 @@ To create and start an ElasticAPM agent use `ElasticAPM.start`:
 ElasticAPM.start(server_url: 'http://localhost:8200')
 ----
 
-  * `config`: A hash or `ElasticAPM::Config` instance with configuration options. See <<configuration,Configuration>>.
+  * `config`: An optional hash or `ElasticAPM::Config` instance with configuration
+  options.  See <<configuration,Configuration>>.
 
 If you are using Rails this is done automatically for you.
```

Comment:
Could you add a reference to the page where you show how to instrument a rack app here?